.github/labeler.yml

```diff
@@ -0,0 +1,7 @@
+app:
+- any: ['src/lightning*/**/*', 'tests/tests_app*/**/*', 'docs/source-app/**/*']
+  all: ['!src/pytorch_lightning/**/*', '!tests/tests_pytorch/**/*', '!docs/source-pytorch/**/*']
+
+pl:
+- any: ['src/pytorch_lightning/**/*', 'tests/tests_pytorch/**/*', 'docs/source-pytorch/**/*']
+  all: ['!src/lightning*/**/*', '!tests/tests_app*/**/*', '!docs/source-app/**/*']
```

Comment:
what does `all` do?

Reply:
`all` here means that "`all`" of these conditions need to be satisfied to be labeled as "pl" in this case.
Examples here: https://github.com/actions/labeler#common-examples

Reply:
gotcha! I didn't see `!`.